dev-libs/libreport/libreport-2.9.5.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_{4,5,6} )
+
+inherit autotools ltprune python-r1 user
+
+DESCRIPTION="Generic library for reporting software bugs"
+HOMEPAGE="https://github.com/abrt/libreport"
+SRC_URI="https://github.com/abrt/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="+gtk"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+COMMON_DEPEND="${PYTHON_DEPS}
+	app-admin/augeas
+	>=dev-libs/glib-2.21:2
```

Comment:
configure says 2.43 is the minimum.

Reply:
Also it seems there is a dependency on systemd for journal support.

Reply:
fixed dev-libs/glib depend.

will explore the systemd depend later.